BraintreeCore/src/main/java/com/braintreepayments/api/PreferredPaymentMethodsCallback.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api
+
+/**
+ * Callback for receiving result of [PreferredPaymentMethodsClient.fetchPreferredPaymentMethods].
+ * This interface is currently in beta and may be removed in future releases.
```

Comment:
We removed this in [V6 of BT iOS](https://github.com/braintree/braintree_ios/pull/932). It was never merchant-facing or documented, so I wonder if we have to wait until BT Android V5 to remove it?

Reply:
In that PR it says that [BTPreferredPaymentMethods.swift](https://github.com/braintree/braintree_ios/pull/932/files#diff-4ed982cde9458d83f20034f0b8c65d3fa8f4f96a1470d25db35214629a17dbc5) was public. Perhaps it was hidden from the merchant in a diferent way? is that possible in iOS?
Still; yes, we ll have to wait for a next major version to completely delete this class, as it is a breaking change

Reply:
Yeh, it was just hidden from the merchant via our reference documentation. It was technically still public & accessible to merchants - just not documented.